classes/SpecificPrice.php

```diff
@@ -441,7 +441,7 @@ protected static function formatIntInQuery($first_value, $second_value)
         $id_customer,
         $id_cart,
         $real_quantity,
-        $id_zone
+        $idZone
```

Comment:
no camelCase for this function please, cf the comment inside the function :)

Reply:
Ok ;) Done!

Reply:
Dang it ! Sorry about that...